src/monitoring/error.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ErrorBoundary } from 'react-error-boundary';
+
+import logger from 'monitoring/logger';
+
+const errorHandler = error => {
+  logger.error(error.message, error.stack);
+};
+
+const ErrorMonitoringProvider = props => {
+  return (
+    <ErrorBoundary
+      FallbackComponent={() => <div>Unexpected error</div>}
```

Comment:
This should be localized, and maybe we should have a design for the error page I will ask Hana about this